### DISCUSSION TITLE
终端输入 npm install hexo-cli -g 显示多个报错，求指教！🙏🏻

### DISCUSSION THREAD

**Discussion by TsingBochun** [hexojs/hexo#5366] [Question / Answer] [answered]
如题，我按照官网教程输入npm install hexo-cli -g后有多个报错，具体请见图片，请教各位这是什么问题？又要如何解决呢？万分感谢！
![截屏2023-11-26 下午5 41 16](https://github.com/hexojs/hexo/assets/5371713/b2a2e3bb-655e-463e-9b31-a9c28d27ff20)

**Comment by stevenjoezhang** [comment]
你的nodejs是用什么装的？我看了一下，如果是homebrew安装的nodejs，默认是不会用`/usr/local/lib/node_modules`这个目录的

**Comment by uiolee** [comment, marked as answer]
Nothing to do with hexo. The log tell you that `npm` can't access to the path.

try:

- delete `/usr/local/lib/node_modules/` or `/usr/local/lib/node_modules/hexo-cli`

or

- uninstall and reinstall `node`
